### PR TITLE
Add cljfmt plugin & travis hook / format sources

### PR DESCRIPTION
**Pull request by rm-hull**
Had problems getting `lein-cljfmt` to play nicely with the travis-supplied lein version; adding `before_install: yes | sudo lein upgrade` to the config seemed to fix it.

Source was bulk formatted with `lein cljfmt fix` in each of the subdirs

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 lein: lein2
-install: (cd cloverage && lein2 deps) && (cd lein-cloverage && lein2 deps)
-script: (cd cloverage && lein2 test) && (cd lein-cloverage && lein2 test)
-sudo: false
-cache:
-  directories:
-    - $HOME/.m2
+sudo: true
+
+before_install: yes | sudo lein upgrade
```
Comment (lvh) [inline .travis.yml]: The `lein` key in travis.yml also supports precise lein versions. That won't update by itself, but it does mean that we can use the `sudo: false` container-based builds, which are faster and updated more regularly.

Comment (lvh) [inline .travis.yml]: (Oh, and I think you need those to enable the cache.)

Comment (rm-hull) [inline .travis.yml]: Just knocked out the sudo: true, and its back to failing: https://travis-ci.org/cloverage/cloverage/jobs/166044926

Comment (lvh) [inline .travis.yml]: Sorry, I wasn't clear. You can specify: `lein: 2.7.1` instead of `lein: lein2` and then you don't need sudo.

Comment (rm-hull) [inline .travis.yml]: ah right, will try that, ... thanks!

**Comment by lvh** [comment]
awesome!

**Pull request merged**